Security Recommendations/Private preview recommendations/TVM-recommendations/SecurityPolicies/TVM-EnableSecureBootWindows-Audit/azurepolicy.rules.json

```diff
@@ -0,0 +1,39 @@
+{
+  "if": {
+    "allOf": [
+      {
+        "field": "type",
+        "equals": "Microsoft.Compute/virtualMachines"
+      },
+      {
+        "field": "Microsoft.Compute/virtualMachines/storageProfile.osDisk.osType",
+        "equals": "Windows"
+      },
+      {
+        "field": "Microsoft.Compute/virtualMachines/storageProfile.imageReference.offer",
+        "in": [
+          "windowsserver-gen2preview-preview"
```

Comment:
is this the only offer for windows ? can you cross check with sirfame's templates ?